spec/requests/monthly_report_spec.rb

```diff
@@ -99,20 +116,40 @@
     let(:report_params) { new_report.attributes.reject { |k, _| k =~ /id\z/ || k =~ /_at/ } }
     let(:patch_params) { { monthly_report: report_params } }
     let(:user_report) { MonthlyReport.find_by(report_params, user: report.user) }
+
     context 'valid' do
       before do
         login report.user
-        patch monthly_report_path report, patch_params
       end
+      context 'when shipped' do
+        before do
+          patch monthly_report_path report, patch_params
+        end
+        after(:each) do
+          ActionMailer::Base.deliveries.clear
+        end
 
-      it { expect(response).to have_http_status :redirect }
-      it { expect(user_report).to be_present }
-    end
+        it { expect(response).to have_http_status :redirect }
+        it { expect(user_report).to be_present }
+        it { expect(ActionMailer::Base.deliveries.size).to eq(1) }
+      end
 
+      context 'when wip' do
+       let(:wip_report) { build(:monthly_report, :wip) }
```

Comment:
__[rubocop]__
* Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)